plugins/inputs/whois/sample.conf

```
@@ -0,0 +1,13 @@
+# Reads whois data and expose as metrics
+[[inputs.whois]]
+  ## List of domains to query
+  domains = ["example.com", "influxdata.com"]
+
+  ## Use Custom WHOIS server
+  # server = "whois.iana.org"
```

Commented out config options should reflect the default value.
```suggestion
  ## Use Custom WHOIS server
  ##   example: server = "whois.iana.org"
  # server = ""
```

---

Fixed